diff --git a/include.js b/include.js
index 78486a71fb428228cf9e9273cc84c90d9c2a42b1..eb1c4c4e086d47b4a109a90864d4f3a98c804e24 100644
--- a/include.js
+++ b/include.js
@@ -1,18 +1,22 @@
 const os = require('os')
 
 const arch = os.arch()
-const platform = os.platform() + (arch === 'arm' ? '-arm' : '')
+const platform = os.platform() + (arch === 'arm' ? '-arm' : '') + (arch === 'arm64' ? '-arm64' : '')
 
 switch (platform) {
   case 'linux':
     require('fuse-shared-library-linux/include')
     break
   case 'darwin':
+  case 'darwin-arm64':
     require('fuse-shared-library-darwin/include')
     break
   case 'linux-arm':
     require('fuse-shared-library-linux-arm/include')
     break
+  case 'linux-arm64':
+    module.exports = require('fuse-shared-library-linux-arm64')
+    break
   case 'win32':
     require('fuse-shared-library-win32/include')
     break
